src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -1,9 +1,13 @@
-// PeerReview.java
+// MathUtils.java
 
```

Comment:
3. Testing:
   - The provided test cases in the "main" method are minimal but sufficient to verify basic functionality. Consider adding more test cases to cover edge cases like integer overflow.